mongo/integration/index_view_test.go

```diff
@@ -633,6 +633,38 @@ func TestIndexView(t *testing.T) {
 		}
 		assert.Nil(mt, cursor.Err(), "cursor error: %v", cursor.Err())
 	})
+	mt.RunOpts("clustered indexes", mtest.NewOptions().MinServerVersion("5.3"), func(mt *mtest.T) {
+		const name = "clustered"
+		clustered := mt.CreateCollection(mtest.Collection{
+			Name:       name,
+			CreateOpts: bson.D{{"clusteredIndex", bson.D{{"key", bson.D{{"_id", 1}}}, {"unique", true}}}},
```

Comment:
```suggestion
			CreateOpts: options.CreateCollection().SetClusteredIndex(bson.D{{"key", bson.D{{"_id", 1}}}, {"unique", true}}),
```

This may resolve test failures on Evergreen.